src/components/EdditMovie/EditMovie.tsx

```diff
@@ -1,8 +1,15 @@
+import style from './EditMovie.module.scss'
 import BasicForm from '../BasicForm/BasicForm'
+import Button from '../Button/Button'
 
-const EditMovie = () => (
-  <div>
-    <h2>EDIT MOVIE</h2>
+interface editMovieProps {
+  onClick: () => void
+}
+
+const EditMovie = ({ onClick }: editMovieProps) => (
+  <div className={style.edit_movie}>
+    <Button classname={'close_button'} onClick={onClick} />
+    <h2 className={style.topic}>Edit MOVIE</h2>
     <BasicForm />
```

Comment:
you need to make your form accept values for each form item. So you'd be able to pass the field values here